core/src/main/scala/org/apache/spark/util/AccumulatorV2.scala

```diff
@@ -165,8 +165,13 @@ abstract class AccumulatorV2[IN, OUT] extends Serializable {
   final protected def writeReplace(): Any = {
     if (atDriverSide) {
       if (!isRegistered) {
+        val printName = if (this.name.isDefined) {
+          s" '${this.name.get}' "
+        } else {
+          " "
+        }
```

Comment:
```suggestion
        val printName = this.name.getOrElse("")
```

Reply:
> 

With the suggested change we get two blanks in the message if the name is not defined.